repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Common.scala

```diff
@@ -61,4 +67,13 @@ object KeystoneV2Common {
 
   case class EndpointsData(json: String, vector: Vector[Endpoint])
 
+  object Endpoint {
+    implicit val endpointReads: Reads[Endpoint] = (
```

Comment:
This is pretty cool. I had originally intended to stick this here just so I could use it in both filters, and to import it into the same scope as the JSON processing calls. However, after some quick research on implicit resolution, I learned that implicits in companion objects to associated types are in scope. So by sticking this here, I don't even have to import it anywhere. Yay!

Here's the Scala documentation about implicits resolution:
https://docs.scala-lang.org/tutorials/FAQ/finding-implicits.html#where-do-implicits-come-from